eng/mgmt/automation/generate.py

```diff
@@ -579,7 +684,10 @@ def main():
     service = get_and_update_service_from_api_specs(api_specs_file, spec,
                                                     args['service'])
     args['service'] = service
-    set_or_increase_version_and_generate(sdk_root, **args)
+    stable_version, current_version = set_or_increase_version_and_generate(
+        sdk_root, **args)
+    compare_with_maven_package(sdk_root, service, stable_version,
+                               current_version)
```

Comment:
Is is possible to factor it to another py file/method? This part might be called multiple times in local to tune the CHANGLOG output/format, and it will be more convenient if it can be called separately.

Reply:
Add a separate executable `changelog.py`